frontend/scripts/react-components/dashboard-element/dashboard-panel/dashboard-panel.container.jsx

```diff
@@ -88,6 +88,11 @@ class DashboardPanelContainer extends React.PureComponent {
     }
   ];
 
+  static propTypes = {
```

Comment:
Should we start using the es7 syntax always?

Reply:
I'm trying to use it when we have class properties